docs/README.md

```diff
@@ -133,56 +132,56 @@ According to your desire, copy one of the examples stated below and paste it in
 
 #### `disclosure-layout` props
 
-| Prop name           | Type                  | Description                                                                                                                                                                                     | Default value                                  |
-| ------------------- | --------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------- |
-| `initialVisibility` | `enum`                | Defines the initial visibility of the layout content. Possible values are: `visible` (content initially open) or `hidden` (content is only displayed with user interaction).    | `hidden`   |
-| `animated`          | `boolean` | Whether or not the layout content should have animations. When set as `true`, this prop will enable additional `data-\*` attributes on the content which you can use as selectors in CSS. It will also ensure that the element will get hidden once the transition has ended. | `false` |
+| Prop name           | Type      | Description                                                                                                                                                                                                                                                                 | Default value |
+| ------------------- | --------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| `initialVisibility` | `enum`    | Defines the initial visibility of the layout content. Possible values are: `visible` (content initially open) or `hidden` (content is only displayed with user interaction).                                                                                                | `hidden`      |
+| `animated`          | `boolean` | Defines if the layout content should have animations. When set as `true`, this prop will enable additional `data-\*` attributes on the content, which you can use as selectors in CSS. It will also ensure that the element will be hidden once the transition has ended. | `false`       |
```

Comment:
```suggestion
| Prop name | Type | Description | Default value |
| - | - | - | - |
| `initialVisibility` | `enum` | Defines the initial visibility of the layout content. Possible values are: `visible` (content initially open) or `hidden` (content is only displayed with user interaction). | `hidden` |
| `animated` | `boolean` | Defines if the layout content should have animations. When set as `true`, this prop will enable additional `data-\*` attributes on the content, which you can use as selectors in CSS. It will also ensure that the element will be hidden once the transition has ended. | `false` |
```